libcxx/CMakeLists.txt

```diff
@@ -481,6 +481,21 @@ include(config-ix)
 include(HandleLibC) # Setup the C library flags
 include(HandleLibCXXABI) # Setup the ABI library flags
 
+# Set C library in use to define respective macro in __config_site
```

Comment:
I would suggest we do this instead (around line 753 where we handle other `__config_site` variables):

```
if (RUNTIMES_USE_LIBC STREQUAL "llvm-libc")
  config_define(_LIBCPP_LIBC_LLVMLIBC 1)
elseif (RUNTIMES_USE_LIBC STREQUAL "picolibc")
  config_define(_LIBCPP_LIBC_PICOLIBC 1)
elseif (RUNTIMES_USE_LIBC STREQUAL "newlib")
  config_define(_LIBCPP_LIBC_NEWLIB 1)
endif()
```

That's more consistent with what we do for the other `__config_site` macros.